tools/@aws-cdk/enum-updater/lib/static-enum-mapping.json

```diff
@@ -1200,6 +1200,24 @@
       "sdk_service": "lambda",
       "sdk_enum_name": "UpdateRuntimeOn",
       "match_percentage": 1
+    },
+    "EventRecordFormat": {
+      "cdk_path": "aws-cdk/packages/aws-cdk-lib/aws-lambda/lib/schema-registry.ts",
+      "sdk_service": "lambda",
+      "sdk_enum_name": "EventRecordFormat",
+      "match_percentage": 1
+    },
+    "KafkaSchemaRegistryAccessConfigType": {
+      "cdk_path": "aws-cdk/packages/aws-cdk-lib/aws-lambda/lib/schema-registry.ts",
+      "sdk_service": "lambda",
+      "sdk_enum_name": "Type",
+      "match_percentage": 1
+    },
+    "KafkaSchemaValidationAttribute": {
+      "cdk_path": "aws-cdk/packages/aws-cdk-lib/aws-lambda/lib/schema-registry.ts",
+      "sdk_service": "lambda",
+      "sdk_enum_name": "Attribute",
```

Comment:
Same for this: https://docs.aws.amazon.com/lambda/latest/api/API_KafkaSchemaValidationConfig.html